samples/src/style.css

```diff
@@ -0,0 +1,51 @@
+html {
```

Comment:
I notice scrollbars in right side whenever I open a sub thingy. Perhaps we should avoid that? Just put the comment in relevant file.
![Screenshot 2024-04-09 at 12 03 42](https://github.com/spacemakerai/forma-extensions-samples/assets/95701996/ad074e7c-a93a-47a7-81f8-1c8f3630c90d)